tests/rsp_test.S

```diff
@@ -104,3 +109,23 @@ command_reset_log:
     # Reset the big log pointer to the start
     jr ra
     sw zero, %lo(BIG_LOG_PTR)
+
+command_big:
+    addi s1, rspq_dmem_buf_ptr, -128
+    move s2, zero
+command_big_loop:
+    lw t0, %lo(RSPQ_DMEM_BUFFER)(s1)
```

Comment:
What about using `CMD_ADDR`?

Reply:
unfortunately that macro only accepts a constant as argument, so you cannot use it in a for loop. If there's a suggestion in this regard, it's well accepted